_data/simple-icons.json

```
@@ -3771,6 +3771,11 @@
             "hex": "000000",
             "source": "https://crystal-lang.org/media/"
         },
+        {
+            "title": "CSDN",
+            "hex": "E85724",
+            "source": "https://yt3.googleusercontent.com/aIZZsltxJ33YrgdAv9-WOYLknclqrr9dE6qhkmXTVPq8f2E62eR_hdpk5ZnWu5oUmixWTfQCDg=s900-c-k-c0x00ffffff-no-rj"
```

Let's use the SVG available instead of manually vectorizing: https://github.com/simple-icons/simple-icons/issues/10811#issuecomment-2336685423.
```suggestion
            "source": "https://www.csdn.net/company/index.html"
```